spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/ReleaseService.java

```diff
@@ -185,9 +188,18 @@ public Release install(Release release) {
 		Map<String, Object> mergedMap = ConfigValueUtils.mergeConfigValues(release.getPkg(), release.getConfigValues());
 		// Render yaml resources
 		String manifestData = ManifestUtils.createManifest(release.getPkg(), mergedMap);
+		DumperOptions dumperOptions = new DumperOptions();
```

Comment:
Wouldn't the ManifestUtils.createManifest be a better place to keeps the  YAML reload/dump gymnastic?